### ISSUE TITLE
Change name of console script to match name of package

### ISSUE THREAD

**Issue by diazona** [diazona/setuptools-pyproject-migration#107]
The `pipx run` command makes it very convenient to download a package and run a script entry point which has the same name as the package. If we want to take advantage of that convenience (and I'd say we do), we should change the name of our console script from `setup-to-pyproject` to `setuptools-pyproject-migration`. (Or at least we need to add a console script with that name, but I don't think there's any point in having two scripts that do the same thing, except for backward compatibility but we don't really have to care about that while the package is at such an early stage of development.)

**Comment by diazona** [comment]
@sjlongland any thoughts? I tentatively assigned this to the next milestone since if we are going to do it, we should do it ASAP (and it'll be an easy change anyway, probably a simple find-and-replace with e.g. `sed`), but I'm definitely not implying that this couldn't be discussed.

**Comment by sjlongland** [comment]
We can add the new name and remove the old one when we get to a 1.0 release (or maybe 0.3/0.4).

If it's a minor change, and the old one is literally just a one-line change to remove, we can defer that… it's not like it's any extra effort on our part. :-)

**Comment by sjlongland** [comment]
I suppose not wanting to pull the rug too quickly.  For certain though, we shouldn't leave it around too long.

Given the alpha status of the project, this isn't as big a problem as it could be, but still, I like to give people just a little warning of a possible breaking change.

**Comment by diazona** [comment]
Yeah, I agree with that, but that seems like a reason to avoid removing the existing script name for a little while, not a reason to avoid adding a new one. It would be fine to have two names for the script for a version or two. (I actually found [another project that does that](https://github.com/kkinder/mehditor) just today.)

In this specific case, I'm also thinking about the fact that we've only had one (non pre-)release, which was not widely advertised, and it doesn't even really work, because of #100. So the negative effects of removing the existing script name are very low compared to most other situations in which we might contemplate a change like this. (And the fact that the project has not reached version 1.0 should be an indicator to people to expect that things can change without warning.)

**Comment by sjlongland** [comment]
Well, it works, just not via the script we intended.  I'm fine either way, the fact we advertise our "major" version as 0 is reason enough to expect the occasional rug-pull.

(Heck, most of mine stay at 0.0.x for years.)

Still, I like to not get into the habit of making breaking changes even at this early stage because it then becomes a hard habit to break out of when a project matures. :-)

**Comment by diazona** [comment]
> Still, I like to not get into the habit of making breaking changes even at this early stage because it then becomes a hard habit to break out of when a project matures. :-)

That's a fair point. I'm not worried about it though because we both agree that this change would be unusual, and not something to make a habit out of. (And if any other regular contributors were to come on board to this project, they would probably pick up on the same thing.)

And yeah, the "it" that doesn't work was the script - I was speaking imprecisely though, sorry!
